buddybuild_carthage_command.sh

```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
 carthage version
+
+brew unlink carthage && HOMEBREW_NO_AUTO_UPDATE=1
+brew install --HEAD https://gist.github.com/jdhealy/940eaac689728068c4179dc93269dc06/raw/fef073f74b9046a2377344ddf129a0f247442ffe/carthage.rb
+
```

Comment:
Is this temporary?

Reply:
yeah, I'm removing it...just using this PR to try to find a solution to the Carthage issue...only the other two files will be the ones changed in this PR